apps/backend/app/api/routes/onboarding.py

```
@@ -174,7 +173,7 @@ async def reset_onboarding(
             "message": "Onboarding progress reset successfully"
         }
         
-    except Exception as e:
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)